xbmc/cores/DataCacheCore.h

```diff
@@ -117,4 +122,12 @@ class CDataCacheCore
     float m_tempo;
     float m_speed;
   } m_stateInfo;
+
+  struct STimeInfo
+  {
+    time_t m_startTime;
+    int64_t m_time;
```

Comment:
yes, this is by intent. if for example the starttime of a stream can't be determined because epg is missing but there is a buffer that allows to rewind to a position before the stream was started.